src/Apps/Sell/Routes/ThankYouRoute.tsx

```diff
@@ -1,41 +1,57 @@
 import * as React from "react"
-import { Message, Spacer, Text } from "@artsy/palette"
-import { graphql, useFragment } from "react-relay"
-import { ThankYouRoute_submission$key } from "__generated__/ThankYouRoute_submission.graphql"
+import { Button, FullBleed, Join, Message, Spacer, Text } from "@artsy/palette"
 import { SubmissionLayout } from "Apps/Sell/Components/SubmissionLayout"
+import { RouterLink } from "System/Components/RouterLink"
+import { AppContainer } from "Apps/Components/AppContainer"
 
-const FRAGMENT = graphql`
-  fragment ThankYouRoute_submission on ConsignmentSubmission {
-    externalId
-  }
-`
-interface ThankYouRouteProps {
-  submission: ThankYouRoute_submission$key
-  children: React.ReactNode
-}
+export const ThankYouRoute: React.FC = () => {
+  return (
+    <FullBleed>
+      <AppContainer>
+        <SubmissionLayout hideNavigation>
+          <Join separator={<Spacer y={4} />}>
+            <Join separator={<Spacer y={2} />}>
+              <Text variant="lg-display">
```

Comment:
[Figma](https://www.figma.com/design/E6rVm1tiSI4PW3EzibKKZw/Submission-%26-Pre-Consignment-Management?node-id=2036-30767&m=dev) 👀 

```suggestion
              <Text variant={["lg-display", "xl"]}>
```

Reply:
We might need to update this for each step or even extract the title component.

Reply:
Will create a separate ticket for it

Reply:
Or just a PR, I don't think it's much work 🐈  Could even be done in this PR.

Reply:
To keep Jira clean and clear 👀

Reply:
I would prefer to create a separate ticket for this because this pr has issues

Reply:
I've written it down and will update the title font later 👍

Reply:
Thank youu, but no neeed, I've just opened a pr https://github.com/artsy/force/pull/14047